plugin/src/main/kotlin/com/vanniktech/maven/publish/ProjectExtensions.kt

```diff
@@ -51,24 +51,16 @@ internal fun Project.javaVersion(): JavaVersion {
   return JavaVersion.current()
 }
 
-internal fun Project.isAtLeastKotlinVersion(
+internal fun Project.isAtLeastKgpVersion(
   id: String,
   major: Int,
   minor: Int,
   patch: Int,
 ): Boolean {
-  val plugin = project.plugins.getPlugin(id) as KotlinBasePlugin
-  val elements = plugin.pluginVersion.takeWhile { it != '-' }.split(".")
-  val kgpMajor = elements[0].toInt()
-  val kgpMinor = elements[1].toInt()
-  val kgpPatch = elements[2].toInt()
-  return kgpMajor > major ||
-    (
-      kgpMajor == major && (
-        kgpMinor > minor ||
-          (kgpMinor == minor && kgpPatch >= patch)
-      )
-    )
```

Comment:
It looks safe to use `kgpMajor >= major && kgpMinor >= minor && kgpPatch >= patch` directly, please tell me if I'm missing something.